bin/aws-kinesis-agent.Ubuntu

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env bash
+
+### BEGIN INIT INFO
+# Provides:          aws-kinesis-agent
+# Required-Start:    $network
+# Required-Stop:     $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Daemon for Amazon Kinesis Agent.
+# Description:       This script is responsible for running Amazon Kinesis Agent
+#                    as a daemon.
+### END INIT INFO
+
+# get init functions: status()
+[ -e /lib/lsb/init-functions ] && . /lib/lsb/init-functions
+
+#set -x
+DAEMON_NAME=aws-kinesis-agent
+NAME=$DAEMON_NAME  # ubuntu compatibility
+AGENT_USER=aws-kinesis-agent-user
+PIDFILE=/var/run/$DAEMON_NAME.pid
+LOG_DIR=/var/log/$DAEMON_NAME
+SHUTDOWN_TIME=11   #10 second default value in AgentConfiguration.java, +1 second buffer
+INITLOGFILE=/tmp/$DAEMON_NAME.`date '+%Y%m%d%H%M%S'`.initlog
+
+# This script is in /etc/rc.d/init.d/ and the executable is in /usr/bin
+BASEDIR=${BASEDIR%/}
+DAEMON_EXEC=$BASEDIR/usr/bin/start-$DAEMON_NAME
+LOGLEVEL=${LOGLEVEL:-INFO}
+ARGS=${ARGS:-}
+RETVAL=0
+
+# load any configs/environment from /etc/defaults/<name>
+[ -e /etc/defaults/$DAEMON_NAME ] && . /etc/defaults/$DAEMON_NAME
+
+# Export the settings
+do_start () {
+  # Return
+  #   0 if daemon has been started
+  #   1 if daemon was already running
+  #   other if daemon could not be started or a failure occured
+  start-stop-daemon --start \
```

Comment:
I'm not familiar with Ubuntu, is "start-stop-daemon" provided by /lib/lsb/init-functions?